docs/doxygen/pages.txt

```diff
@@ -71,6 +71,9 @@ Some configuration settings are C pre-processor constants, and some are function
 @section MQTT_MAX_CONNACK_RECEIVE_RETRY_COUNT
 @copydoc MQTT_MAX_CONNACK_RECEIVE_RETRY_COUNT
 
+@section MQTT_DO_NOT_USE_CUSTOM_CONFIG
```

Comment:
IMO, it would be better to have this config at the top of the file to set the context about the importance of the custom config and the config defaults (from both the perspective of reading the `core_mqtt_config_defaults.h` file and the doxygen output)

Reply:
sure